packages/account/src/Sections/Security/ConnectedApps/data-list-template.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { ApplicationObject } from '@deriv/api-types';
+import { Button } from '@deriv/components';
+import { toMoment } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import DataListTemplateEntry from './data-list-template-entry';
+import { getConnectedAppsScopes } from './template-helper';
+
+type TDataListTemplate = { data_source: ApplicationObject; handleToggleModal: (app_id: number) => void };
```

Comment:
Question: in file [connected-apps-revoke-modal.tsx](https://github.com/binary-com/deriv-app/pull/10314/files#diff-d2c8cfa7858f56b24a5746578cbccf70ea8835ba16b73ddf654dea68c1bde661) noticed that `app_id` can be `undefined | null | number` 
Here we're making it `number` only, are they related? 
If yes, then there will be a type difference error